app/code/Magento/Customer/Model/Session.php

```diff
@@ -175,9 +182,11 @@ public function __construct(
             $cookieMetadataFactory,
             $appState
         );
+        $this->_eventManager->dispatch('customer_session_init', ['customer_session' => $this]);
```

Comment:
Why you move it above?

Reply:
Sort in alphabetical order ;) I can revert this change if you request it. Thanks

Reply:
It’s better to keep only needed changes in order to prevent conflicts, so please revert it.

Reply:
**done**